core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -852,7 +852,13 @@ public PropServerConfiguration(
             if (loadAdditionalConfigurations && httpServerEnabled) {
                 this.jsonQueryConnectionCheckFrequency = getInt(properties, env, PropertyKey.HTTP_JSON_QUERY_CONNECTION_CHECK_FREQUENCY, 1_000_000);
                 this.jsonQueryFloatScale = getInt(properties, env, PropertyKey.HTTP_JSON_QUERY_FLOAT_SCALE, 4);
```

Comment:
nit: is it a good idea to increase the default scales to 10 and 19, i.e. the max values, or something higher than the current 4 and 12? I recall users complaining about too low default scale from time to time.

Reply:
OK, let's keep them as is.